scripts/enelespacio.js

```diff
@@ -8,7 +8,7 @@
 //   None
 //
 // Commands:
-//   hubot en el espacio - Muestra quiénes están en el espacio (fuera de la tierra) AHORA
+//   huemul en el espacio - Muestra quiénes están en el espacio (en el International Space Station o ISS) AHORA
```

Comment:
En los comando se debe usar el keyword `hubot`, no `huemul`

Reply:
tenia esa duda hace tiempo y en mis scripts de huemul ponia `huemul` y no `hubot` porque son propios y no external-scripts. No compartes eso?

Reply:
Es independiente de si es interno o externo. Según la documentación esa es la forma https://hubot.github.com/docs/scripting/#documenting-scripts

Reply:
pero es nuestro hubot a eso voy
sino, voto por sacarlos todos `#   <trigger> - <what the hear trigger does>`

Reply:
De hecho los cambie todos a hubot

Reply:
ok cambio